src/graphics/sprite.rs

```diff
@@ -12,6 +12,11 @@ use std::{
     ptr::{self, NonNull},
 };
 
+/// !! WARNING !!
```

Comment:
I wouldn't put such a dramatic warning here.

Maybe `Note:` would be better.

Reply:
Changed to Note: